doc/reference/autoinstall-reference.rst

```diff
@@ -1133,9 +1133,10 @@ Example:
 .. code-block:: yaml
 
    autoinstall:
-     # Disable SSH root login and start the ufw firewall automatically
+     # Make sure history files are purged if one runs `apt purge etckeeper`
+     # Also start the ufw firewall automatically.
      debconf-selections: |
-       openssh-server openssh-server/permit-root-login boolean false
```

Comment:
While this is an improvement to drop the incorrect text here, I think we have a problem where people may have picked up this incorrect knowledge and come back here looking for an answer.

What I want to do is to correct the example and offer a short explanation with details on why it works the way it does, along with a link to the underlying behavior.

Reply:
That's fair, hopefully we can get rid of this example later (either if we implement https://bugs.launchpad.net/subiquity/+bug/2129466 or in a couple of years maybe).

Updated.